common.py

```diff
@@ -1005,10 +1005,37 @@ def apply_rt_signalregion(cols):
     cols.cutflow['rt_signalregion'] = len(cols)
     return cols
 
-def apply_cutbased(cols):
+def check_if_model_exists(model_file, xrootd_url) :
+    # Check if the file exists locally
+    if not os.path.exists(model_file):
+        print(f"File {model_file} not found. Downloading from {xrootd_url}...")
+        try:
+            os.makedirs(os.path.dirname(model_file), exist_ok=True)  # Ensure directory exists
+            subprocess.run(["xrdcp", xrootd_url, model_file], check=True)
+            print(f"Downloaded {model_file} successfully.")
+        except subprocess.CalledProcessError as e:
+            print(f"Error downloading {model_file}: {e}")
+            return None
+
+def apply_cutbased(cols, lumi, ddt_map_file = 'models/cutbased_ddt_map_ANv6.json', xrootd_url = 'root://cmseos.fnal.gov//store/user/lpcdarkqcd/boosted/cutbased_ddt/') :
```

Comment:
we should keep `apply_cutbased()` as the non-DDT version. make a separate `apply_cutddt()` (or something) for this version. and propagate this to the CR/anti selections introduced by #12

Reply:
Added for the anti selections, I was a bit hesitant to do this for the CR. I think it depends on how we want to use the DDT and wasn't sure how much sense it made to do it for the CR.

Reply:
the low-ecfM2 CR indeed does not fully make sense with DDT, I think.